### ISSUE TITLE
In ssr mode all templates are compiled in template folder ?

### ISSUE THREAD

**Issue by geminigeek** [Dave136/vue-email#80]
hi,

i am setting it up like here https://www.vuemail.net/getting-started/ssr , i have three templates all of them are compiling while i just want to wok on one "WelcomeEmail" ? here is the output when i set verbose true .

```

💌 Generating output
🚧 Compiling TailwindEmail file
🚧 Compiling VerifyEmail file
🚧 Compiling WelcomeEmail file
🌎 Injecting translations
🎉 Rendering template WelcomeEmail
```

this is my config 

```
const vueEmail = config("./templates", {
  verbose: true,
  
})
```

**Comment by Flowko** [comment]
yeah, thats because now, we auto import all email components, and they need to be compiled before being used in ur email templates,  we'll try to resolve it later, to only use if u have any of them on the template you are rendering

**Comment by Flowko** [comment]
with verbos enabled, it will just log all activites, if its annoying you, you can turn it off

**Comment by geminigeek** [comment]
I dont have issue with logs, I am just wondering, why it's compiling all templates , I plan to use many templates in that folder ,it might have performance issue , that's my only concern

**Comment by Flowko** [comment]
yep true, we'll see how to resolve that later on
